deploy/azuredeploy.json

```diff
@@ -307,6 +307,13 @@
             "metadata": {
                 "description": "A CIDR notation IP for Docker bridge."
             }
+        },
+        "aksEnableRBAC": {
+            "type": "bool",
+            "defaultValue": false,
```

Comment:
Is there a reason why we want RBAC to be disabled by default ? 
I would suggest to make it default. 

Please see this cluster hardening guide on using RBAC :
https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster

Reply:
That is a good question. I think this point did come up in discussion with the AKS folks and if I understand correctly the default for AKS right now is RBAC disabled by default and would therefore make sense to keep the default value in the helm chart to be disabled as well.